tests/installation/partitioning_raid.pm

```diff
@@ -304,38 +328,41 @@ sub run {
         addraid(2);
 
         assert_screen 'partition-format';
-        send_key "alt-s";
+        send_key(is_storage_ng() ? 'alt-a' : "alt-s");
         assert_screen 'partitioning_raid-filesystem-focused';
-        send_key 'down' for (1 .. 3);
-        assert_screen 'partitioning_raid-filesystem_ext4';
-        send_key "alt-m";
+        send_key 'down';
+        send_key 'home';
+        send_key_until_needlematch 'partitioning_raid-filesystem_ext4', 'down';
+        send_key 'alt-o' if is_storage_ng;
+        send_key 'alt-m';
         assert_screen 'partitioning_raid-mount_point-focused';
         type_string "/boot";
         assert_screen 'partitioning_raid-mount_point-_boot';
 
-        send_key $cmd{finish};
+        send_key(is_storage_ng() ? $cmd{next} : $cmd{finish});
         if (get_var('LVM')) {
-            assert_screen 'partitioning_raid-raid_ext4_added-lvm';
+            send_key_until_needlematch 'partitioning_raid-raid_ext4_added-lvm', 'down';
         }
         else {
-            assert_screen 'partitioning_raid-raid_ext4_added';
+            send_key_until_needlematch 'partitioning_raid-raid_ext4_added', 'down';
         }
     }
 
     # select RAID add
-    send_key $cmd{addraid};
+    send_key(is_storage_ng() ? 'alt-d' : $cmd{addraid});
     assert_screen 'partitioning_raid-menu_add_raid';
     setraidlevel(0);    # RAID0 for swap
     assert_screen 'partitioning_raid-raid_0-selected';
     addraid(1);
 
     # select file-system
     assert_screen 'partition-format';
-    send_key $cmd{filesystem};
+    send_key(is_storage_ng() ? 'alt-a' : $cmd{filesystem});
     assert_screen 'partitioning_raid-filesystem-focused';
-    send_key "end";
-    assert_screen 'partitioning_raid-swap_format-selected';
-    send_key $cmd{finish};
+    #send_key "end";
+    #assert_screen 'partitioning_raid-swap_format-selected';
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit